test/elixir/test/attachments_test.exs

```diff
@@ -109,6 +109,11 @@ defmodule AttachmentsTest do
     resp = Couch.delete("/#{db_name}/bin_doc/foo.txt", query: %{w: 3})
 
     assert resp.status_code == 409
+    
+    resp = Couch.delete("/#{db_name}/bin_doc/notexisting.txt?rev=1-a0f595984a7c4cf1dafc4cf4c04081d6")
+    assert resp.status_code == 404
+    assert resp.body["error"] == "not_found"
```

Comment:
I wonder why we don't see the "Document is missing attachment" error reason propagate here...

Also, we probably shouldn't hardcode the revision number but get the current on from the document (`rev` variable should have?).

Reply:
Fixed in https://github.com/apache/couchdb/pull/3667/commits/0b62354ba5c10ec5a891e1525b828e76939cb2c9 and https://github.com/apache/couchdb/pull/3667/commits/c2bc260de0aafc5c8d16116899641c62559c6747.